strings/british_talk.json

```diff
@@ -0,0 +1,118 @@
+{
+
+    "words": {
+        "id": "BBM Pin",
```

Comment:
This is stupid.

Reply:
wtf how could you be so offensive

Reply:
Jesus Christ bacon that reference is ANCIENT

Reply:
back in my day.... based

Reply:
Should be 'loicense'